dbmodel/ddls.go

```diff
@@ -53,11 +53,13 @@ var TableCreateSQL = []string{
 	`CREATE TABLE IF NOT EXISTS Subscription (
 		  ID int(11) NOT NULL,
 		  ProductID int(11) NOT NULL,
+		  SubscriptionCode int(11) NOT NULL AUTO_INCREMENT,
```

Comment:
okay, if the subscription code is primary key, and you want to auto increment, then lets just keep it that way. but, `ProductID` should be a string. Since subscription code is auto_inc, we dont need it in the util.go struct to create it.